docs-en/chapter_preface/suggestions.md

```diff
@@ -2,223 +2,225 @@
+     * Multiline
+     * comments
      */
     ```
 
 === "C"
 
     ```c title=""
-    /* Header comments for labeling functions, classes, test samples, etc. */
+    /* Header comments for labeling functions, classes, test samples, etc */
    
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
     /**
-     * multi-line
-     * marginal notes
+     * Multiline
+     * comments
      */
     ```
 
 === "Zig"
 
     ```zig title=""
-    // Header comments for labeling functions, classes, test samples, etc.
+    // Header comments for labeling functions, classes, test samples, etc
    
-    // Content comments for detailed code solutions.
+    // Content comments for details.
    
-    // Multi-line
-    // Annotation
+    // Multiline
+    // comments
     ```
 
-## Learn Efficiently In Animated Graphic Solutions
+## Efficient Learning via Animated Illustrations
 
-Compared with text, videos and pictures have a higher degree of information density and structure and are easier to understand. In this book, **key and difficult knowledge will be presented mainly in the form of animations and graphs**, while the text serves as an explanation and supplement to the animations and graphs.
+Compared with text, videos and pictures have a higher density of information and are more structured, making them easier to understand. In this book, **key and difficult concepts are mainly presented through animations and illustrations**, with text serving as explanations and supplements.
 
-If, while reading the book, you find that a particular paragraph provides an animation or a graphic solution as shown below, **please use the figure as the primary source and the text as a supplement and synthesize the two to understand the content**.
+When encountering content with animations or illustrations as shown in the figure below, **prioritize understanding the figure, with text as supplementary**, integrating both for a comprehensive understanding.
 
-![Example animation](../index.assets/animation.gif)
+![Animation Illustration Example](../index.assets/animation.gif)
 
-## Deeper Understanding In Code Practice
+## Deepen Understanding through Code Practice
 
-The companion code for this book is hosted in the [GitHub repository](https://github.com/krahets/hello-algo). As shown in the figure below, **the source code is accompanied by test samples that can be run with a single click**.
+The source code of this book is hosted on the [GitHub Repository](https://github.com/krahets/hello-algo). As shown in the figure below, **the source code comes with test examples and can be executed with just a single click**.
 
-If time permits, **it is recommended that you refer to the code and knock it through on your own**. If you have limited time to study, please read through and run all the code at least once.
+If time permits, **it's recommended to type out the code yourself**. If pressed for time, at least read and run all the codes.
```

Comment:
"If time permits, **it's recommended to type out the code yourself**. If pressed for time, at least read and run all the codes."
can be changed to ->
"If time allows, **it's recommended to manually type out the code**. In case of time constraints, ensure to at least read and run all the provided code snippets."